plasmapy/dispersion/core.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Since this file is empty, we don't need to add it at this point.  I know it was in the outline we discussed in the last meeting, but it was really just there as a visual placeholder.  So, go ahead and delete it, and if we find functionality in the future for it then we can bring it back.